wandb/integration/keras/callbacks/metrics_logger.py

```diff
@@ -119,6 +122,7 @@ def on_batch_end(self, batch: int, logs: Optional[Dict[str, Any]] = None) -> Non
             wandb.log(logs)
 
             self.global_batch += self.log_freq
+            wandb.log({})
```

Comment:
let's condition it to `if isinstance(self.log_freq, int):`.

The idea is - when the user is logging epoch wise (log_freq == epoch) the system metrics will be logged such that the x axis is representative of the epoch. If the user is logging batch wise the system metrics x-axis should be batch steps.

Reply:
@ayulockin can you explain what you mean here? The system metrics are logged independently and could not be related to the epoch, the x axis for them is the wall time.

Reply:
It was a misunderstanding on our part. :)

Reply:
The system metrics are logged as expected.